site/pages/docs/ref/multimedia/multimedia-fr.hbs

```diff
@@ -30,6 +30,16 @@
 	</ul>
 </section>
 
+<section>
+<h2>Évaluation et rapport</h2>
+<ul>
+	<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-audio-en.html">Anglais Audio Seulement Rapport</a></li>
+	<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-audio-fr.html">Français Audio Seulement Rapport</a></li>
+	<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-video-en.html">Anglais Video Rapport</a></li>
+	<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-video-fr.html">Français Video Rapport</a></li>
```

Comment:
Le caractère `&` dois être échapé. `&amp;`

Reply:
Addressed in commit "Updated assessment format"